src/projectscene/view/clipsview/clipcontextmenumodel.cpp

```diff
@@ -29,14 +32,28 @@ void ClipContextMenuModel::load()
         return item;
     };
 
+    auto makeInternalItem = [this](const ActionCode& actionCode) {
+        muse::ui::UiAction action;
+        action.code = actionCode;
+        action.title = muse::TranslatableString("action", "Rename clip");
```

Comment:
I think the function should either be called `makeRenameClipItem` or take a name